### ISSUE TITLE
1.0.0: Changes on client.id / client.sessionId ?

### ISSUE THREAD

**Issue by endel** [colyseus/colyseus#241]
Hi everyone, I've been considering this for a long time and would like to hear your thoughts on this. 

TLDR: I want to change the way `client.id` works, and make `client.id` act as the `client.sessionId`. Making `client.sessionId` deprecated on 1.0.0 (with warnings, not completely removed)

---

### 👉 What's your experience with [`client.id`](https://docs.colyseus.io/server/client/#id-string), [`client.sessionId`](https://docs.colyseus.io/server/client/#sessionid-string)? Have you used `client.id` to achieve something you couldn't have with `client.sessionId`?

---

The `client.id` is a unique id that's meant to be used to identify the same client between rooms.

**Pros:** 
- not much effort to identify the same user between rooms

**Cons:**
- not reliable, can be bypassed easily by using Incognito mode in the browsers
- can't be stored in the database, since the same client may have a different `id` once the `localStorage` is cleared in the client-side.

The `client.sessionId` on the other hand is used to correlate data in the state to a particular client. (e.g. a `Player` entity, etc.)

**Comment by marwan38** [comment]
So far, I've only used session id to map the state to the client. Probably wouldn't make a difference to me considering my use case.

**Comment by ArnolFokam** [comment]
Right now i'm experiencing a problem when using client.sessionId . When deploying the app on an offline server and connecting to the app with another computer locally ,My colyseus monitor  shows two connection per client.id with the different sessionId. This gives me unpredictable results. I think a work around would be to use client.id for player identification rather than sessionId.

**Comment by endel** [comment]
The `client.id` has been soft deprecated - it is now an alias to `client.sessionId`. See the migration guide for `0.11` here: https://docs.colyseus.io/migrating/0.11/
